### PR TITLE
Use mrh build for CI

### PR DESCRIPTION
**Pull request by dkubb**

### CODE REVIEW

**Comment by dkubb** [comment]
@mbj this is ready for review.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,49 @@
+FROM dkubb/alpine-ruby
+MAINTAINER Dan Kubb <dan@bloomcrush.com>
+
+ENV TZ=utc \
+  RAILS_ENV=test \
+  RACK_ENV=test \
+  NEW_RELIC_ENV=test \
+  JRUBY_OPTS='--2.0 -J-Xmx1g --debug' \
```
Comment (mbj) [inline Dockerfile]: Do we need the `JRUBY_OPTS` here where we do not officially support jruby?

Comment (dkubb) [inline Dockerfile]: I think you're right. I will remove this.

At some point I would like to test jruby out, but not now.

**Comment by dkubb** [comment]
@mbj this is ready for another pass.

**Pull request merged**